Makefile

```diff
@@ -30,8 +30,8 @@ HONCHO_MANAGE_TESTS := honcho -e .environ/.env.test run python3 manage.py
 
 # Parameters ##################################################################
 
-# For `test_one` use the rest as arguments and turn them into do-nothing targets
-ifeq ($(firstword $(MAKECMDGOALS)),$(filter $(firstword $(MAKECMDGOALS)),test_one manage))
+# For `test.one` use the rest as arguments and turn them into do-nothing targets
+ifeq ($(firstword $(MAKECMDGOALS)),$(filter $(firstword $(MAKECMDGOALS)),test.one manage))
```

Comment:
If you change it here, change it also on Ocim, for consistency. I would simply leave it as is, unless there is a good reason to change it?

Reply:
The changes I made throughout this file were truly out of desiring a better experience with having to type these over and over, comparing this style with underscores and a new lingo "migration_*autogen*" to the make targets in edX docker devstack. I would actually be *very* cool with opening a PR for this. I can do it as part of https://github.com/open-craft/opencraft/pull/254 which I hope to get to soon.

Reply:
OK, but in the future when you make changes on one side, can you make them on the other side at the same time? Otherwise we'll inevitably forget some.

Reply:
Alrighty, I'll pick up that PR and make the changes on Friday :+1: I've wanted to do it so no worries.

Reply:
OK, perfect, thanks!

Reply:
@UmanShahzad Did you end up doing this?

Reply:
@antoviaque Ugh, embarrassingly no. I ended up way over my planned hours for the week, so I decided to not pursue more work. To be sure I do this, I'm making a ticket for this upcoming sprint.